.github/workflows/cd.yml

```diff
@@ -20,10 +23,11 @@ jobs:
       - uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
         with:
           node-version: 18
+          registry-url: https://registry.npmjs.org
+      - run: npm install -g npm
       - run: npm ci
       - name: Publish
         run: |
-          npm config set //registry.npmjs.org/:_authToken ${NPM_TOKEN}
-          npm publish --ignore-scripts --access public
+          npm publish --provenance --ignore-scripts --access public
```

Comment:
What does the `provenance` flag do?

Reply:
It will help us to sign the packages in npm, so it will appears as (in the UI):
![image](https://user-images.githubusercontent.com/5110813/233311911-adeec4a9-d472-4056-bc4b-16d384264aa4.png)

More context in https://github.blog/2023-04-19-introducing-npm-package-provenance/